languages/translations/ru.json

```diff
@@ -62,49 +62,57 @@
                 "default": "Вы прибыли в {nth} пункт назначения",
                 "upcoming": "Вы прибудете в {nth} пункт назначения",
                 "short": "Вы прибыли",
-                "short-upcoming": "Вы скоро прибудете"
+                "short-upcoming": "Вы скоро прибудете",
+                "named": "Вы прибыли в {waypoint_name}"
```

Comment:
The correct token to use here is probably either `{waypoint_name:date}` or `{waypoint_name:accusative}`, but I’d defer to @yuryleb. We can also fix this afterwards, once the strings are in Transifex.

Reply:
Actually no, grammar rules work only with street names but `waypoint_name` could be any name :sad: So it's much easy just to change source strings to force nominal case for names than to extend grammar rules to handle any names.

Reply:
Good point, I didn’t really consider how the open-endedness could break the grammar rules’ assumptions. Since the client knows more about the waypoint name’s semantics, it could conceivably call `grammarize()` depending on whether it’s a street name or something else, but that wouldn’t happen by default. So I agree that putting the name in an appositive, as you suggested in https://github.com/Project-OSRM/osrm-text-instructions/pull/235#discussion_r180550563, neatly sidesteps the problem. I think we’ll need to do the same thing for at least one other language anyways.